compute/gpu-instances.rst

```diff
@@ -45,10 +45,11 @@ A GPU instance :doc:`is launched <launching-an-instance/index>` the same way as
 
 Installing the driver
 ---------------------
+In order to use the GPU functionality, a driver from NVIDIA needs to be installed on the instance that has access to the virtual GPU. Please follow below instructions to install the driver.
 
-In order to use the GPU functionality, a driver from nVidia needs to be installed on the instance that has access to the virtual GPU. Please follow below instructions to install the driver.
-
-The current version of the driver that we support is: **535.129.03**.
+The current NVIDIA vGPU Software Version that we are running is: **16.4**
+The current latest driver we support is **Linux:535.161.05 Windows:538.33** 
```

Comment:
says 535.161.05 but the download says 535.161.07

Reply:
Thanks, 

Have fixed and pushed new correct version.